context-free.md

```diff
@@ -1,22 +1,56 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+The language is targeted toward anyone with access to a computer (Windows,
+Mac, or Linux).  
```

Comment:
This is a little broad - a good way to approach this is to look at the design decisions made while developing the language. If you can answer why X is hard, but Y is easy, and look at who benefits from Y being easy, then you can figure out the target audience. You can also work backwards - given that Z was the target audience, it makes sense for Y to be easy.